src/aktualizr_secondary/aktualizr_secondary_interface.h

```diff
@@ -0,0 +1,29 @@
+#ifndef AKTUALIZR_SECONDARY_INTERFACE_H
+#define AKTUALIZR_SECONDARY_INTERFACE_H
+
+#include "uptane/manifest.h"
+
+class IAktualizrSecondary {
```

Comment:
What's the benefit of having an interface that only the Secondary uses? If it's just for the mock, couldn't the mock inherit directly from the AktualizrSecondary class?

Reply:
Yes, at the moment it's just for the mock/test. Yes, the mock can inherit from AktualizrSecondary. I've considered a few options including the one you mentioned. The drawback of this approach is a circular reference between AktualirzSecondary and AktualizrSecondaryMsgDispatcher (which can be mitigated by a forward declaration)  and the test becomes depending on AktualizrSecondary implementation internals, which can be mitigated by applying PIMPL to AktualizrSecondary. Frankly speaking, I don't have a strong opinion here, the inheritance also works for me as long as a proper forward declaration is applied and MsgDispatcher and AktualizrSecondaryMsgDispatcher are declared in different header files so SecondaryTcpServer does not depend on AktualizrSecondaryMsgDispatcher.

Reply:
OK, I will leave it unchanged in this PR, anyway it looks like I will have to rework it in order to implement the new install method (my current task).